.travis.yml

```diff
@@ -1,19 +1,24 @@
 language: python
-
 sudo: false
-
 python:
-  - '3.4'
-  - '3.5'
-  - '3.6'
-
+- '3.4'
+- '3.5'
+- '3.6'
 install:
-  - pip install flake8 pytest-cov pytest coveralls
-  - pip install -e .
-
+- pip install flake8 pytest-cov pytest coveralls
+- pip install -e .
 script:
-  - pytest --cov=trustymail
-  - flake8 .
-
+- pytest --cov=trustymail
+- flake8 .
 after_success:
-  - coveralls
+- coveralls
+deploy:
```

Comment:
I'm not an expert on TravisCI, but this looks correct.  

One question:  Does TravisCI know to update the `__version__` string in `__init__` appropriately?

Reply:
No, Travis will push out a tagged commit: https://docs.travis-ci.com/user/deployment/pypi/#Deploying-tags